nishimweelysee/Elysee-CapstoneProject
# Ft user portfolio page

#### What does this PR do?
>- This PR display the portfolio of the website owner and i have took it like articles. it contains the projects that describe > >   what i can do.
>
#### Description of Task to be completed
>- Project Image
>- project Title
>- Read more button which link to the project online
>
#### How should this be manually tested?
>Open a website and navigate to Portfolio page in navigation Menu you will see a list of all portfolio posted.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
>Ft(User blog Page) Use this Link https://trello.com/c/FiFsqqRh to view my trello board.
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92327183-2cf0fa80-f058-11ea-938b-aac67a2be5e0.png)

Thank you!!
